src/Native/pkg/runtime.native.System.IO.Compression/win10/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -4,12 +4,12 @@
   
   <PropertyGroup>
     <Version>4.0.1</Version>
-    <PackageTargetRuntime>win10-$(PackagePlatform)</PackageTargetRuntime>
+    <PackageTargetRuntime>win10-$(PackagePlatform)-aot</PackageTargetRuntime>
   </PropertyGroup>
 
   <ItemGroup>
     <NativeFile Include="$(WinNativePath)..\..\NetCoreForCoreCLR\native\clrcompression.dll">
-      <TargetPath>runtimes/win10-$(PackagePlatform)/lib/netcore50</TargetPath>
+      <TargetPath>runtimes/win10-$(PackagePlatform)-aot/lib/netcore50</TargetPath>
```

Comment:
We could change this back to native now, but I'm ok leaving as is.  Either way this package will need an update once CoreRT starts using the *aot* monikers.  CoreRT will be outside of AppX.  I expect by that time we'll have the native+tfm feature in NuGet and we can refactor to use that.